### ISSUE TITLE
Moonplus dont seems to be compatible with moonscript anymore?

### ISSUE THREAD

**Issue by CriztianiX** [pigpigyyy/MoonPlus#17]
Trying to get a minimal testing, moonplus dont seems to be compatible with moonscript anymore?

```
lapis  = require "lapis"
class extends lapis.Application
  "/": => json: { status: true }
```
Gives me as response

```
{
  "status": true
}

```

But compiling with moonp, the same code, it gives me an error

```
2020/08/05 13:49:50 [error] 18#18: *64 lua entry thread aborted: runtime error: error loading module 'app' from file './app.lua':
 	./app.lua:21: '}' expected (to close '{' at line 16) near '__parent'
 stack traceback:
 coroutine 0:
 	[C]: in function 'require'
 	/usr/local/openresty/luajit/share/lua/5.1/lapis/init.lua:15: in function 'serve'
 	content_by_lua(nginx.conf.compiled:30):2: in main chunk, client: 172.18.0.1, server: , request: "POST / HTTP/1.1", host: "127.0.0.1:29875"
```

**Comment by CriztianiX** [comment]
This is the moonc output
https://pastebin.com/u4d3UNRN

This is the moonp output
https://pastebin.com/U6kXawY7

**Comment by daelvn** [comment]
This is the `moonp` output for the snippet, as seen in the Pastebin:

```lua
local lapis = require("lapis")
do
	local _class_0
	local _parent_0 = lapis.Application
	local _base_0 = {
		["/"] = function(self)
			return {
				json = {
					status = true
				}
			}
		end
	}
	_base_0.__index = _base_0
	setmetatable(_base_0, _parent_0.__base)
	_class_0 = setmetatable({ -- LINE 16
		__init = function(self, ...)
			return _class_0.__parent.__init(self, ...)
		end,
		__base = _base_0 -- LINE 20
		__parent = _parent_0
	}, {
		__index = function(cls, name)
			local val = rawget(_base_0, name)
			if val == nil then
				local parent = rawget(cls, "__parent")
				if parent then
					return parent[name]
				end
			else
				return val
			end
		end,
		__call = function(cls, ...)
			local _self_0 = setmetatable({}, _base_0)
			cls.__init(_self_0, ...)
			return _self_0
		end
	})
	_base_0.__class = _class_0
	if _parent_0.__inherited then
		_parent_0.__inherited(_parent_0, _class_0)
	end
	return _class_0
end
```

As you can see, there is a comma missing after `_base_0` at Line 20, in the class definition. This looks like an undetected compile error.

**Comment by pigpigyyy** [comment]
Thanks for issuing the compiler error. The bug is fixed in e4f0bc2e930cc9fa0adc1d52f5f706db92166f41.
